plugins/rapid7_intsights/plugin.spec.yaml

```diff
@@ -461,16 +559,19 @@ actions:
         description: Original value
         type: string
         required: true
+        example: "apple.com"
```

Comment:
Maybe we should use "example.com", but I don't know the plugin context, so if not please ignore my comment.

Reply:
That's a good point. I'll update this.